front/app/containers/IdeasShowPage/index.tsx

```diff
@@ -68,36 +50,23 @@ const StyledIdeasShow = styled(IdeasShow)`
   `}
 `;
 
-const StyledSignInButton = styled(Button)`
-  margin-bottom: 20px;
-`;
-
 const IdeasShowPage = () => {
   const { slug } = useParams() as { slug: string };
   const idea = useIdea({ ideaSlug: slug });
   const { windowWidth } = useWindowSize();
   const tablet = windowWidth <= viewportWidths.tablet;
 
+  if (isUnauthorizedError(idea)) {
+    return <Unauthorized />;
+  }
+
   if (isError(idea)) {
-    return (
-      <IdeaNotFoundWrapper>
-        <p>
-          <FormattedMessage {...messages.sorryNoAccess} />
-        </p>
-        <StyledSignInButton
-          linkTo="/sign-up"
-          text={<FormattedMessage {...messages.signUp} />}
-        />
-        <Link to="/sign-in">
-          <FormattedMessage {...messages.signIn} />
-        </Link>
-      </IdeaNotFoundWrapper>
-    );
+    return <PageNotFound />;
   }
 
-  if (!isNilOrError(idea)) {
+  if (idea) {
     return (
-      <Container>
+      <Box background="white">
```

Comment:
Change to `colors.white` just in case we decide to use a slightly different shade of white at some point.